### PR TITLE
json.from_file

### PR DESCRIPTION
**Pull request by jorivanee**
Personally I read JSON from a file very frequently, and its honestly quite annoying how there isn't a built in function for this

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by JustMaffie** [comment]
Personally I don't know how the python documentation works, so this still needs documentation.

**Comment by merwok** [comment]
Hello!  The links in the previous message go to all the useful information about the development process.
In particular, pull requests should be opened in relation to a ticket on bugs.python.org, sometimes after a discussion on the python-ideas mailing list (where you can see if many other people want the same thing, or if they point you to an existing way of doing what you need).

File: Lib/json/__init__.py [hunk 1]
```diff
@@ -272,6 +272,16 @@ def detect_encoding(b):
     return 'utf-8'
 
 
+def from_file(filepath, *, cls=None, object_hook=None, parse_float=None,
+        parse_int=None, parse_constant=None, object_pairs_hook=None, **kw):
+    with open(filepath) as _data:
+        data = _data
```
Comment (chadmiller-amzn) [inline Lib/json/__init__.py:278]: What's going on here?! You copy the file object out, then the `with` closes it (hopefully), then you try to read from the closed file?

**Review by chadmiller-amzn — CHANGES_REQUESTED** [review]

**Comment by merwok** [comment]
Please discuss this new feature proposal on python-ideas and reopen a pull request if there is agreement on its usefulness!

**Pull request closed without merging**